runtime/bcverify/staticverify.c

```diff
@@ -1316,20 +1322,20 @@ checkStackMap (J9CfrClassFile* classfile, J9CfrMethod * method, J9CfrAttributeCo
 						}
 
 						if ((frameType >= CFR_STACKMAP_APPEND_1) && (frameType <= CFR_STACKMAP_APPEND_3)) {
-							maxForEntries = code->maxLocals;
+							checkAppendArraySize = TRUE;
```

Comment:
Why is this it only necessary to check this on append frames?

Reply:
I think this can be more general to entries with local variable arrays. I will verify that is the case.